config/sets/symfony/symfony43.php

```diff
@@ -27,6 +27,11 @@
             RenameMethodRector::METHOD_CALL_RENAMES => ValueObjectInliner::inline([
                 new MethodCallRename('Symfony\Component\BrowserKit\Response', 'getStatus', 'getStatusCode'),
                 new MethodCallRename('Symfony\Component\Security\Http\Firewall', 'handleRequest', 'callListeners'),
+                new MethodCallRename(
```

Comment:
Could you link the rename in Symfony with a comment here? Just so we know the origin of this change in Symfony

```php
// @see https://...
```

Reply:
Done that :)

Reply:
Thanks :+1: 

What about its sister?

https://github.com/symfony/http-kernel/blob/801b925e308518ddf821ba91952c41ae77c77507/Event/GetResponseForExceptionEvent.php#L67

Reply:
Good call! Fixed that, too :)